### PR TITLE
Editorial pass to simplify text and unify terminology.

### PR DESCRIPTION
**Pull request by chris-wood**
This is mainly editorial, so there should be nothing surprising here. Please let me if that's not the case!

It also includes clarifying text per @sayrer's recommendation raised during WGLC.

cc @jhoyla @martinthomson

### CODE REVIEW

File: draft-ietf-tls-external-psk-importer.md [hunk 1]
```diff
@@ -87,20 +86,24 @@ when, and only when, they appear in all capitals, as shown here.
...
+identities if imported incorrectly. Endpoints may incrementally deploy PSK Importer support
 by offering non-imported keys for TLS versions prior to TLS 1.3. Non-imported and imported PSKs
 are distinct since their identities are different on the wire. See {{rollout}} for more details.
 
-Clients which import external keys TLS MUST NOT use these keys for any other purpose.
-Moreover, each external PSK MUST be associated with at most one hash function.
+Clients which import external keys MUST NOT use either the external keys or the derived
+keys for any other purpose. Moreover, each external PSK MUST be associated with at most
+one hash function, as per the rules in Section 4.2.11 from {{!RFC8446}}.
+See {{security-considerations}} for more discussion.
```
Comment (chris-wood) [inline draft-ietf-tls-external-psk-importer.md]: @sayer this should address your comment. Please let me know if not!

Comment (sayrer) [reply draft-ietf-tls-external-psk-importer.md:106]: WFM

File: draft-ietf-tls-external-psk-importer.md [hunk 2]
```diff
@@ -87,20 +86,24 @@ when, and only when, they appear in all capitals, as shown here.
...
-based on some contextual information before use in a connection. In contrast to key exporters,
-wherein differentiation is done via an explicit label and context string, the key importer
-defined herein diversifies external one PSK into one or more PSKs for use via a target protocol,
-KDF identifier, and optional context string. Additionally, the resulting PSK binder key is modified
-with a new derivation label to prevent confusion with non-imported PSKs.
-
-Imported keys do not require negotiation for use, as a client and server will not agree upon
-identities if not imported correctly. Endpoints may incrementally deploy PSK importer support
+The PSK Importer interface mirrors that of the TLS Exporters interface in that
+it diversify a key based on some contextual information. In contrast to the Exporters
```
Comment (davidben) [inline draft-ietf-tls-external-psk-importer.md:90]: s/diversify/diversifies/

File: draft-ietf-tls-external-psk-importer.md [hunk 3]
```diff
@@ -155,47 +156,47 @@ PSK is being imported. For example, TLS 1.3 {{!RFC8446}} and QUICv1 {{!QUIC=I-D.
...
+derived key is necessary.
 
-EPSKs may be imported for early data use if they are bound to protocol settings and configurations that would
-otherwise be required for early data with normal (ticket-based PSK) resumption. Minimally, that means ALPN,
-QUIC transport settings, etc., must be provisioned alongside these EPSKs.
+EPSKs may be imported before the start of a connection if the target KDFs, protocols, and
+context string(s) are known a priori. EPSKs may also be imported for early data use
+if they are bound to protocol settings and configurations that would otherwise be
+required for early data with normal (ticket-based PSK) resumption. Minimally, that
+means ALPN, QUIC transport settings, etc., must be provisioned alongside these EPSKs.
```
Comment (davidben) [inline draft-ietf-tls-external-psk-importer.md:193]: This is an existing sentence, but the word "minimally" here is interesting given PSKs meant for use with non-QUIC connections don't need QUIC transport settings (nitpick: I think QUIC calls them "transport parameters").

Comment (chris-wood) [reply draft-ietf-tls-external-psk-importer.md]: Good catch -- fixed!

**Pull request merged**